<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="button_tampil_zakatmasuk">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Tampil Form Zakatmasuk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_zakat">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Tampil Form Zakat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>FORM UTAMA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_zakatkeluar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Tampil Form Zakatkeluar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_muzaki">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Tampil Form Muzaki</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tampil_mustahik">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Tampil Form Mustahik</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
